### DISCUSSION TITLE
Commonsense Knowledge, Symbolic Knowledge Distillation paper: Why is there more diversity after filtering?

### DISCUSSION THREAD

**Discussion by mayanks43** [eduardoblanco/csc696H-Spring24#11] [Q&A]
"As we filter, we surprisingly observe proportionally more diverse examples." What intuitions do the authors give for observing more diversity after filtering? Do you agree with it? What else could be happening?

**Comment by eduardoblanco** [comment]
@hyunssong Ideas?

**Comment by hyunssong** [comment]
The authors intuition was that filtered examples have a similar sentence structure -- hence, removing this will give more diversity. Some doubt I had about this intuition is that logical misalignments or awkward phasing (which the authors claim to be the two types of filtered examples) don't necessarily seem to have similar sentence structures. I suspect the way how they do the diversity analysis could be the underlying reason of the increased diversity. Maybe showing more diversity measures or showing the removed examples actually have high similarity will be more convincing?
